<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>1114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="MainUser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>561</width>
      <height>951</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="MainFriends">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Friends</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visit Profile</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="MainSuggFriends">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Suggested Friend</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visit Profile</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Add Friend</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="LogOut">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="UserFriends">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Friends</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>View Profile</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Add Friend</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Home">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UserBack">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="UsersName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>521</width>
       <height>611</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>609</height>
       </rect>
      </property>
      <widget class="QLabel" name="MainPosts">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>481</width>
         <height>371</height>
        </rect>
       </property>
       <property name="text">
        <string>User Posts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SingInScreen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>561</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Social Network Sign In</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="NamePrompt">
       <property name="text">
        <string>Welcome! Please enter your Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="NameInput">
       <property name="text">
        <string>Aled Montes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="SignIn">
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
